runtime/bcutil/ROMClassBuilder.hpp

```diff
@@ -37,6 +37,12 @@
 #include "ClassFileParser.hpp"  /* included to obtain definition of VerifyClassFunction */
 #include "StringInternTable.hpp"
 
+#define MAX_INTERFACE_INJECTION 2
+typedef struct InjectionInfo{
+	J9UTF8 *interfaces[MAX_INTERFACE_INJECTION];
+	U_16 numOfInterfaces;
+} InjectionInfo;
```

Comment:
@tajila 
Should I add a boolean to represent injection, or is it fine to continue to check if injection is needed by doing _injectionInfo->numOfInterfaces > 0?

Reply:
`_injectionInfo->numOfInterfaces > 0` is fine